### DISCUSSION TITLE
Re-stream of RTSP with any filter throws error

### DISCUSSION THREAD

**Discussion by rockstar2020** [bluenviron/mediamtx#1026] [Questions]
Hello,
First of all, amazing project. Well done!
I have the latest version of rtsp-simple-server installed in a docker container on Ubuntu 22.04.
The setup went smoothly and I have several cameras configured with no issues.
I have a doorbell camera which has a fisheye lens.
Using the following ffmpeg command (in Ubuntu) with lens filter I can easily correct the video stream and re-stream it to a path defined in rtsp-simple-server configuration:
_ffmpeg -re -i rtsp://user:password@xxx.xxx.xxx.xxx:554/H.264 -vf lenscorrection=cx=0.5:cy=0.5:k1=-0.400:k2=0.020 -f rtsp rtsp://IP_of_rtsp-simple-server:8554/doorbell_

But when I add the same command as runOnInit in configuration like below I get an error message in the log and as a result the re-stream doesn't work. 

**My configuration:**
_paths:
  doorbell:
  original:
    source: rtsp://user:password@xxx.xxx.xxx.xxx:554/H.264
    sourceProtocol: tcp
    runOnInit: ffmpeg -re -i rtsp://localhost:$RTSP_PORT/$RTSP_PATH -vf lenscorrection=cx=0.5:cy=0.5:k1=-0.400:k2=0.02 -f rtsp rtsp://localhost:$RTSP_PORT/doorbell
    runOnInitRestart: yes_

**Error in the log:**
_2022/07/11 21:59:04 INF [path original] runOnInit command exited with code 0_

Can you point me to the right direction to troubleshoot this issue?

Many thanks in advance

**Comment by aler9** [comment]
Hello, ffmpeg is not included inside the Docker image, please read here: https://github.com/aler9/rtsp-simple-server/discussions/278#discussioncomment-549104

**Reply by rockstar2020** [reply]
Thanks a lot!
It now makes sense why it doesn't work. :)

[a comment by an automated account is not shown]
